cartography/intel/cve/feed.py

```diff
@@ -98,6 +98,9 @@ def _call_cves_api(url: str, api_key: str | None, params: Dict[str, Any]) -> Dic
             retries += 1
             if retries >= MAX_RETRIES:
                 raise
+            # Exponential backoff
+            sleep_time *= 2
+            time.sleep(sleep_time)
```

Comment:
so a total of 3 * 2^5 or 6 * 2^5 seconds when API key is provided?

That seems low. Do we need higher number?

Reply:
why   2^5?
condition at L99 is >= , so it will actually iterate over 6. 
3 * 2^6 = 192
3 * 2^6 = 384
I think those times are high enough. 👀

Reply:
heh, let's try to enumerate it.
it's incremented by 1 before the check and stopped when the value reaches 6
```
retries variable value -> sleep time
1 -> 3
2 -> 6
3 -> 12
4 -> 24
5 -> 48
6 -> stopped (hit >= MAX_RETRIES)
```